scheduler/src/cook/authorization.clj

```diff
@@ -144,6 +145,47 @@
                      "is not allowed to perform" verb "on" (str object ",") "denying.")
            false))))
 
+(defn admins-open-gets-auth
+  "Allows admins to do anything to any object, and if whitelisted-user is truthy,
+  allows non-admins to manipulate objects that they own and to read (get) any object."
+  [admins user verb {:keys [owner] :as object} whitelisted-user]
```

Comment:
`whitelisted-user` implies it contains a user name. Rename it to something like `user-passed-whitelist?`